src/app/pages/home/tutor/tutor.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './tutor.scss';
+
+function Buttons({data}) {
+    return (
+        <div className="buttons">
+            <a className="btn primary" href={data.buttonLink}>{data.buttonText}</a>
+            <a className="btn" href={data.demoLink}>{data.demoText}</a>
+        </div>
+    );
+}
+
+function IconGroup({icon, text}) {
+    return (
+        <div className="icon-group">
+            <img src={icon} alt />
```

Comment:
should `alt` have a value?  Seems strange to have an empty attribute like this

Reply:
Images that don't contribute meaning to the page are supposed to have empty alts so the screenreader doesn't include them. I've gone back and forth between `alt` and `alt=""`

Reply:
oh cool, I did not know that.  Good to learn!